src/store/modules/substance.js

```diff
@@ -125,13 +127,29 @@ const getters = {
       privateQCNote: detail.attributes.privateQcNote,
       publicQCNote: detail.attributes.publicQcNote
     };
+  },
+  staticState: staticState => {
+    let { detail } = staticState;
+    return {
+      preferredName: detail.attributes.preferredName,
+      displayName: detail.attributes.displayName,
+      casrn: detail.attributes.casrn,
+      qcLevel: detail.relationships.qcLevel.data.id,
+      source: detail.relationships.source.data.id,
+      substanceType: detail.relationships.substanceType.data.id,
+      description: detail.attributes.description,
+      privateQCNote: detail.attributes.privateQcNote,
+      publicQCNote: detail.attributes.publicQcNote
+    };
+    
   }
 };
 
 // mutations
 const mutations = {
   ...rootMutations,
   loadDetail(state, payload) {
+    this.staticState = JSON.parse(JSON.stringify(payload));
```

Comment:
Apparently `JSON.parse` can have some issues handling some data types (like NaN, undefined, empty). Since our data is 'clean' `JSON.parse` may still be a good option.

Alternative is lodash `this.staticState = _.cloneDeep(payload);`